README.md

```diff
@@ -30,8 +30,11 @@ console.log(one) // [1]
 swap(one, -1, 0)
 console.log(one) // [1]
 
-function swap(array, first_index, second_index) {
-  // -- YOUR ANSWER HERE --
+function swap(arr, index_1, index_2) {
+  let temp = [];
+  temp.index_1 = arr.index_1;
+  arr.index_1 = arr.index_2
+  arr.index_2 = arr.index_1
```

Comment:
How do you access an item in an `array` that is at index `3`?
Does `temp` not need to be an array?  How many values are you ever storing in it?
Please test, re-write, and test again to check that you are getting the results you expect.